src/AuthModule/Components/Login/Login.tsx

```diff
@@ -1,9 +1,8 @@
 import { useForm } from "react-hook-form";
 import logo from "../../../assets/PMS 3.svg";
-import React, { useContext, useState } from "react";
+import  { useContext, useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
 import axios from "axios";
-// import { toast } from "react-toastify";
 import Loader from "../../../SharedMoudule/Components/Loading/Loading";
 import { AuthContext } from "../../../Context/Components/AuthContext";
```

Comment:
I noticed using the context for storing constant values such as `baseUrl`, however context is not the right place to store constant values. We can have a constants file in which we store all the links that we need in our application. and this will only be a line import for example:
`import {baseUrl} from "constats/urls";`
and then use it directly instead of importing `useContext` API and the context we need to use and then getting the value that we need from it. This makes the code look more complicated than it needs to be.